README.md

```diff
@@ -23,6 +23,16 @@ Have something to add or change? Open a [pull request](https://github.com/barbea
 - **EGNOS Toolkit** ([Download](https://sourceforge.net/projects/libegnos/files/), [Source code](https://sourceforge.net/projects/libegnos/))-  A set of tools to work with Satellite-Based Augmentation Systems (SBAS), specially EGNOS. Support Linux.
 - **GNSS-SDR** ([Download](https://gnss-sdr.org/), [Source code](https://github.com/gnss-sdr/gnss-sdr)) - An open source GNSS software defined receiver. Supports Linux, Mac, and Windows.
 - **RTKLIB** ([Download](http://www.rtklib.com/), [Source code](https://github.com/tomojitakasu/RTKLIB)) - An Open Source Program Package for GNSS Positioning. Has a companion Android app [RTKGPS+](https://play.google.com/store/apps/details?id=gpsplus.rtkgps). Supports Windows.
+- **gAGE Lab** [Download](https://gage.upc.es/gLAB/) A free and publicly available
+software for GNSS data processing.
+
+### Proprietary tools
+
+- [Jason Positioning-as-a-Service](https://jason.rokubun.cat) An online service
```

Comment:
Could you please describe which services are free for users of any GNSS receiver, and which require payment or purchase of a specific receiver?

Reply:
Thanks Sean for your comment, I've pushed a commit commenting on this. I will perform a pull request to accept this change. Best!